### PR TITLE
Show setup for local user

### PR DESCRIPTION
**Pull request by Havvy**
Also removes the fluff about multiple files, which I don't think is generally useful?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +29,18 @@ system config:
 }
 ```
 
-In reality, my package overrides are split across multiple files, since I have
-more than just this one, but the idea is the same. It should also be possible to
-set this up with a user-local nixpkgs config.
+#### Local User Setup
+
+You can also set this up specifically for a single user by adding it to that
+user's nix configuration, located at `/home/%user%/.nixpkgs/config.nix`.
```
Comment (solson) [inline README.md]: I think saying `$HOME/.nixpkgs/config.nix` or `~/.nixpkgs/config.nix` would be more idiomatic.

Comment (Havvy) [reply README.md]: Went with $HOME.

**Review by solson — APPROVED** [review]
Thanks!

**Comment by Havvy** [comment]
Welcome!

**Comment by solson** [comment]
I've added you as a collaborator on the repo.

**Pull request merged**